modules/oe_content_department/config/install/field.field.rdf_entity.oe_department.oe_department_links.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.rdf_entity.oe_department_links
+    - rdf_entity.rdfentity.oe_department
+  module:
+    - link
+id: rdf_entity.oe_department.oe_department_links
+field_name: oe_department_links
+entity_type: rdf_entity
+bundle: oe_department
+label: Links
```

Comment:
I believe this field got renamed to  "News from this department"

Reply:
Hm...maybe this should be actually a reference to Announcement CTs..